filter/ArticleMedraXmlFilter.inc.php

```diff
@@ -192,6 +192,41 @@ function createArticleNode($doc, $pubObject) {
 			$this->appendTextMiningCollectionNodes($doc, $articleNode, $article, $submissionGalleys);
 		}
 
+		$issueId = $article->getCurrentPublication()->getData('issueId');
+		if ($cache->isCached('issues', $issueId)) {
+		    $issue = $cache->get('issues', $issueId);
+		} else {
+		    $issueDao = DAORegistry::getDAO('IssueDAO'); /* @var $issueDao IssueDAO */
+		    $issue = $issueDao->getById($issueId, $context->getId());
+		    if ($issue) $cache->add($issue, null);
+		}
+		$journalId = $issue->getData('journalId');
+		if(!empty($journalId)){
+		    $accessRights = null;
+		    $journalDao = DAORegistry::getDAO('JournalDAO'); /** @var JournalDAO $journalDao */
+		    $journal = $journalDao->getById($journalId);
```

Comment:
There is already object $context -- which is journal in OJS -- so no need to get it again.

Reply:
There is already object $context which is journal in OJS so no need to get it again.